src/app/lib/layoutGrid.js

```diff
@@ -135,18 +134,17 @@ export const layoutGridItemMedium = css`
 export const layoutGridItemSmall = css`
   ${gelGridMargin}
 
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    grid-column: 1 / span 6;
+  grid-column: 1 / span 6;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```

Comment:
This doesnt match zeplin. At 768px zeplin spans 4 columns but this spans 5

Reply:
Let's look at it with UX tomorrow.